namenode/src/main/java/org/apache/crail/namenode/BlockStore.java

```diff
@@ -222,6 +303,33 @@ short removeDatanode(DataNodeInfo dn) throws IOException {
 
 	//---------------
 
+	public long getTotalCapacity() {
```

Comment:
The naming does not clearly state that this is number of blocks. Maybe `getTotalNumberOfBlocks` or `getTotalNumberBlocks` or `getTotalBlockCount`?